AnnoyancesFilter/sections/popups.txt

```diff
@@ -87,6 +87,7 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+||transparentcalifornia.com/static/js/slickModal.min$script
```

Comment:
Did you find any other modals which are triggered by this script (like login etc.)? I haven't so far (else dangerous rule).

Reply:
Didn't find any useful elements.